cloudbuild.yaml

```diff
@@ -1,10 +1,15 @@
 # See https://cloud.google.com/cloud-build/docs/build-config
 steps:
-- name: gcr.io/cloud-builders/docker
-  entrypoint: /usr/bin/make
-  args:
-    - push-images
-    - TAG=$_GIT_TAG
+  - name: gcr.io/cloud-builders/docker
+    entrypoint: "/usr/bin/make"
+    args:
+      - "push-images"
+      - "TAG=$_GIT_TAG"
+  - name: "gcr.io/google.com/cloudsdktool/cloud-sdk"
+    entrypoint: "/usr/bin/make"
+    args:
+      - "deploy"
+      - "TAG=$_GIT_TAG"
```

Comment:
We could introduce an order of steps to make sure the images are built before deployment. (Also we may have separate cloudbuild files). 
Ben's feedback can be really useful.

Reply:
I'm not sure what needs to be changed, currently it does build and push the image before deploying it to the staging environments

Reply:
Something like that https://cloud.google.com/build/docs/configuring-builds/configure-build-step-order#examples. 

Feel free to ignore my comment if it's not needed.

Reply:
My experience of using CloudBuild is that the jobs run consecutive, though it'll be pretty trivial to add this to ensure that always run in order.

Reply:
Added in 1012a65